packages/flutter_tools/lib/src/project.dart

```diff
@@ -8,19 +8,29 @@ import 'dart:convert';
 
 import 'android/gradle.dart' as gradle;
 import 'base/file_system.dart';
+import 'bundle.dart' as bundle;
 import 'cache.dart';
 import 'flutter_manifest.dart';
 import 'ios/xcodeproj.dart' as xcode;
 import 'plugins.dart';
 import 'template.dart';
 
-/// Represents the contents of a Flutter project at the specified [directory].
+/// Represents the contents of a Flutter project at the specified [projectPath].
```

Comment:
`projectPath` is a parameter of `FlutterProject.fromPath`, not a property of `FlutterProject`.

Reply:
Right!